iiod/ops/factory_ops.cpp

```diff
@@ -57,6 +72,20 @@ AbstractOps* FactoryOps::buildOps(const char* type, std::vector<const char*>& ar
 			return nullptr;
 		}
 		iiodOpsAbstract = new GenericXmlContext(args);
+
+		const char* last_item = args.back();
+		if (is_number(last_item)) {
+			unsigned long i_port = std::stoul(last_item);
```

Comment:
could you document in the commit message at least how and where should this port be set ?